docs/UserGuide.md

```diff
@@ -115,20 +115,22 @@ Examples:
 
 ### Locating elderly by name: `find`
```

Comment:
Might want to change this to Locating elderly by name or nric.

Reply:
Oh yeah my bad